website/client/components/shops/market/equipmentSection.vue

```diff
@@ -116,9 +118,38 @@ layout-section(:title="$t('equipment')")
         return this.marketGearCategories.filter(c => c.id === this.selectedGroupGearByClass)[0];
       },
       sortedGearItems () {
+        let result = this.filterGearItems();
+        let selectedSortKey = sortGearTypeMap[this.selectedSortGearBy.id];
+
+        result = _sortBy(result, selectedSortKey);
```

Comment:
hi @trvrfrd , sorry for the delay here. The code looks good, I'm just wondering if it's possible to condense these 3 sorting functions (2 sortBy and the reverse) into just one since with many items it would be a very expensive operation.

For example you could save the call to reverse() by using orderBy instead of sortBy and change to `desc` as the third parameter in case `selectedSortKey !== 'type' && selectedSortKey !== 'value' === true`

Something like this should work:

```
let order = selectedSortKey !== 'type' && selectedSortKey !== 'value' ? 'desc' : 'asc';
result = _orderBy(result, [selectedSortKey, locked], ['', 'asc'];
```

Reply:
Now I'm sorry for the delay on my end! I've been dealing with something of a family emergency all week, I'll take a look at your feedback and incorporate it ASAP. Thanks @paglias!

Reply:
no worries @trvrfrd ! hope everything's all right